### DISCUSSION TITLE
Thought about rack space colors

### DISCUSSION THREAD

**Discussion by dreng** [netbox-community/netbox#6331] [General]
Hi there,

When I just had a look at the current racks I noticed one of them having a utilization of about 90%. Therefore the color of the bar that indicates the remaining space is red instead of green.

In general, red is something bad, an alert, something you should take care of immediately. If you think like a technican that might be correct, although I think that 90% utilization is something you should know about but no reason to freak out. If you have a look at the situation under economic aspects, 90% is not even enough. Like every machine a rack should be considered most economical if it's being utilized 100%. Thus, red seems to be not an ideal color.

Not a big thing. I just wanted to know what you think about it.

**Comment by jeremystretch** [comment]
We could just use one color regardless of the utilization. That might be more appropriate.

**Comment by baldgeek** [comment]
Just my two cents, but how much work is it to make it user can assign 
colors? I use color from rack utilization since we allocate racks to 
departments. Red tells me we likely need to allocate them another rack. 
I am probably a small set of users, but if people who don't want the 
colors to be different, just set them the same.

**Reply by jeremystretch** [reply]
> Just my two cents, but how much work is it to make it user can assign
colors?

Relative to the value returned, _a lot_.

**Comment by dreng** [comment]
As different colors seems to be useful for some users and the red bar doesn't harm anyone I'd say leave it like it is (or just choose a different color that's not blue or red).
